sdk/eventgrid/Azure.Messaging.EventGrid/tests/EventGridClientTests.cs

```diff
@@ -2,156 +2,29 @@
-                    new SharedAccessSignatureCredential(sasToken),
-                    options));
-            await sasTokenClient.PublishEventsAsync(GetEventsList());
-        }
-
-        private IList<EventGridEvent> GetEventsList()
-        {
-            List<EventGridEvent> eventsList = new List<EventGridEvent>();
-
-            for (int i = 0; i < 10; i++)
-            {
-                eventsList.Add(
-                    new EventGridEvent(
-                        Recording.Random.NewGuid().ToString(),
-                        $"Subject-{i}",
-                        "hello",
-                        "Microsoft.MockPublisher.TestEvent",
-                        Recording.Now,
-                        "1.0"));
-            }
-
-            return eventsList;
-        }
-
-        private IList<EventGridEvent> GetEventsWithTopicsList()
-        {
-            List<EventGridEvent> eventsList = new List<EventGridEvent>();
-
-            for (int i = 0; i < 10; i++)
-            {
-                eventsList.Add(
-                    new EventGridEvent(
-                        Recording.Random.NewGuid().ToString(),
-                        $"Topic-{i}",
-                        $"Subject-{i}",
-                        "hello",
-                        "Microsoft.MockPublisher.TestEvent",
-                        Recording.Now,
-                        "1",
-                        "1.0"));
-            }
-
-            return eventsList;
-        }
-
-        private IList<CloudEvent> GetCloudEventsList()
-        {
-            List<CloudEvent> eventsList = new List<CloudEvent>();
-
-            for (int i = 0; i < 10; i++)
-            {
-                eventsList.Add(
-                    new CloudEvent(
-                        Recording.Random.NewGuid().ToString(),
-                        $"Subject-{i}",
-                        "record",
-                        "1.0"));
-            }
-
-            return eventsList;
-        }
-
-        private IList<object> GetCustomEventsList()
-        {
-            List<object> eventsList = new List<object>();
-
-            for (int i = 0; i < 10; i++)
-            {
-                eventsList.Add(new TestEvent()
-                {
-                    dataVersion = "1.0",
-                    eventTime = Recording.Now,
-                    eventType = "Microsoft.MockPublisher.TestEvent",
-                    id = Recording.Random.NewGuid().ToString(),
-                    subject = $"Subject-{i}",
-                    topic = $"Topic-{i}"
-                });
-            }
+            EventGridClient sasTokenClient = new EventGridClient(
+                new Uri("https://exampletopic.westus2-1.eventgrid.azure.net/api/events"),
+                new SharedAccessSignatureCredential("thisIsNotAFakeCredential"));
 
-            return eventsList;
+            Assert.That(() => sasTokenClient.BuildSharedAccessSignature(DateTimeOffset.UtcNow.AddMinutes(60)),
```

Comment:
nit: parameter indendation - use new line for each param within BuildSharedAccessSig